airbyte-cdk/python/airbyte_cdk/sources/declarative/date/min_max_date.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import datetime
+
+from airbyte_cdk.sources.declarative.interpolation.interpolated_string import InterpolatedString
+
+
+class MinMaxDate:
+    """
+    Compares the provided date against optional minimum or maximum times. If date is earlier than
+    min_date, then min_date is returned. If date is greater than max_date, then max_date is returned.
+    """
+
+    def __init__(
+        self,
+        date: str,
+        datetime_format: str = "%Y-%m-%dT%H:%M:%S.%f%z",
```

Comment:
Not sure if the existing CDK has a default time format we should adhere to

Reply:
rfc 3339 is a good default and is what json schema expects